modules/openapi-generator/src/main/java/org/openapitools/codegen/OpenAPINormalizer.java

```diff
@@ -1251,6 +1251,12 @@ private Schema processNormalize31Spec(Schema schema, Set<Schema> visitedSchemas)
             schema.getTypes().remove("null");
         }
 
+        // process const
```

Comment:
This is the actual fix that transforms a single const into an enum with one value, so the 3.1 and 3.0 representations are equivalent